.github/workflows/ci.yml

```diff
@@ -1,124 +1,159 @@
+                options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php-versions:
+                    - '7.4'
+                    - '8.0'
+
         steps:
-            -   uses: actions/checkout@v2
-
-            -   name: Setup PHP
-                uses: shivammathur/setup-php@v2
-                with:
-                    php-version: ${{ matrix.php }}
-                    extensions: mbstring, intl
-                    tools: composer:v2
-
-            -   name: 'Run Elasticsearch'
-                uses: elastic/elastic-github-actions/elasticsearch@master
-                with:
-                    stack-version: 7.6.0
-
-            -   name: Install eco-ci
-                run: git clone https://github.com/spryker-eco/eco-ci.git ecoci
-
-            -   name: Execute test script
-                run: ./ecoci/build/ga.sh ${{ matrix.composer_preference }}
-    test_mariadb:
-        name: "Test / MariaDB"
-        runs-on: ubuntu-18.04
+            - name: Checkout@v2
+              uses: actions/checkout@v2
+
+            - name: Setup PHP ${{ matrix.php-versions }}
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-versions }}
+                  extensions: mbstring, intl, bcmath, pdo_mysql
+
+            - name: Configure sysctl limits
+              run: |
+                  sudo swapoff -a
+                  sudo sysctl -w vm.swappiness=1
+                  sudo sysctl -w fs.file-max=262144
+                  sudo sysctl -w vm.max_map_count=262144
+            - name: Composer get cache directory
+              id: composer-cache
+              run: |
+                  echo "::set-output name=dir::$(composer config cache-files-dir)"
+            - name: Composer cache
+              uses: actions/cache@v2
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+                  restore-keys: |
+                      ${{ runner.os }}-composer-
+
+            - name: Composer validate
+              run: composer validate
+
+            - name: Composer version
+              run: composer --version
+
+            - name: Composer install
+              run: composer install --prefer-dist --no-interaction --optimize-autoloader
+
+            - name: PHP syntax validation
+              run: find ./src -path src -prune -o -type f -name '*.php' -print0 | xargs -0 -n1 -P4 php -l -n | (! grep -v "No syntax errors detected" )
+
+            - name: Run codeception tests
+              run: composer test
+
+            - name: Run PHPStan checks
+              run: composer stan
+
+            - name: Check codestyle checks
+              run: composer cs-check
+
+    prefer-lowest:
+        runs-on: ubuntu-20.04
```

Comment:
Above you are using version 18.04 ... why 20.04 over here?

Reply:
Fixed.